### PR TITLE
Adds all mint icons to the storybook

### PR DESCRIPTION
**Pull request by AkshayNaikGroww**
## What does this PR do?
Adds all mint icons to the storybook

## What packages have been affected by this PR?
Story book

## Types of changes

What types of changes does your code introduce to this project?

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)

- [ ] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist before merging

_Put an `x` in the boxes that apply_

- [ ] These changes have been thoroughly tested.

- [ ] Changes need to be immediately published on npm.

### CODE REVIEW

File: packages/ui-toolkit/stories/IconStore.stories.tsx [hunk 1]
```diff
@@ -67,3 +68,28 @@ export const AllMaterialIcons = () => {
...
 };
+
+
+export const AllMintIcons = () => {
+  return (
+    <div className="story_icon_all_image_list">
+      {
+        Object.keys(mintIcons).map(icon => {
+          const IconComponent = mintIcons[icon];
+          const importCode = `import { ${icon} } from '@groww-tech/icon-store/mi';`;
```
Comment (saloni-groww) [inline packages/ui-toolkit/stories/IconStore.stories.tsx:79]: shouldn't it be `@groww-tech/icon-store/mint-icons` ?

File: packages/ui-toolkit/.storybook/main.js [hunk 2]
```diff
@@ -5,5 +5,8 @@ module.exports = {
   "addons": [
     "@storybook/addon-links",
     "@storybook/addon-essentials"
-  ]
+  ],
```
Comment (saloni-groww) [inline packages/ui-toolkit/.storybook/main.js:8]: Why are we adding this ?

**Review by saloni-groww — APPROVED** [review]

File: packages/ui-toolkit/stories/IconStore.stories.tsx (hunk 1 above)
Comment (AkshayNaikGroww) [reply packages/ui-toolkit/stories/IconStore.stories.tsx]: corrected

File: packages/ui-toolkit/.storybook/main.js (hunk 2 above)
Comment (AkshayNaikGroww) [reply packages/ui-toolkit/.storybook/main.js:8]: This is needed for running the storybook at node v <16.20

**Review by prashant-adhikari-groww — APPROVED** [review]

**Pull request merged**